### DISCUSSION TITLE
Issue with GitHub Copilot VScode extension, is there an outage?

### DISCUSSION THREAD

**Discussion by atomey** [community/community#127488] [Copilot Conversations]
### Select Topic Area

Question

### Body

edit: reboot fixed it but all the traditional troubleshooting steps (Except reinstall VSCode...) so I don't know why it broke or why it's working again...

I'm having issues with Copilot completions. I haven't made any changes to my configuration, it was working yesterday and even earlier today. Suddenly stopped working. I tried the Chat and initially it was giving responses, but now even the chat responses seem to be slow. This makes me suspect an outage with GH Copilot.

I know this isn't my network because I haven't dropped a single packet to Google and everything else on the same is fine, tried to reboot VSCode, same problem.
Also checked status here:
https://www.githubstatus.com/

Did not see any incidents posted yet. I'm accessing from South Asia (Philippines).

**Comment by Akash1134** [comment]
Closed as resolved
